spinalcordtoolbox/types.py

```diff
@@ -274,6 +274,18 @@ def __init__(self, points_x=None, points_y=None, points_z=None, deriv_x=None, de
         if self.compute_init_distribution:
             self.compute_vertebral_distribution(disks_levels=self.disks_levels, label_reference=self.label_reference)
 
+    # NB: Prior to https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4010, the get_centerline() function
+    #     would return two 3xN arrays: `arr_ctl` and `arr_ctl_der`. These arrays are almost identical to
+    #     `Centerline.points` and `Centerline.derivatives`, except transposed. So, to preserve backwards-compatibility,
+    #     we provide some properties that mimic the old behavior.
+    @property
+    def arr_ctl(self):
```

Comment:
Not sure if the name is appropriate. If I am not wrong, it means "array control" and control is related to a specific technique for generating the centerline. I would replace "arr_ctl" with "points" or "positions" or "centers" or anything more appropriate. The same comment goes for "arr_ctl_der".

Reply:
I am actually wrong, "ctl" probably means "centerline". However, it proves that it is confusing.

Reply:
I agree that the `ctl` variable name isn't obvious at first. 

The reason I used it, however, was to preserve the existing conventions. (`ctl` is present throughout SCT's codebase -- almost everywhere that `get_centerline` is called.)

![image](https://user-images.githubusercontent.com/16181459/218837567-250bf015-4b53-4bc9-b670-fa0cca9e2dda.png)